test/functional/wallet_import_rescan.py

```
@@ -21,6 +21,10 @@
 
 from test_framework.test_framework import BitcoinTestFramework
 from test_framework.governance import EXPECTED_STDERR_NO_GOV_PRUNE
+from test_framework.address import (
+    AddressType,
```

Ruff (0.12.2)

`test_framework.address.AddressType` imported but unused

Remove unused import: `test_framework.address.AddressType`

(F401)